content/ecosystem/bridges/discord/bridges.toml

```diff
@@ -89,3 +89,30 @@ editing = true
 reactions = true
 presence = false
 typing_notifications = true
+
+[[bridges]]
+name = "Out Of Your Element"
+maintainer = "Cadence"
+summary = "Modern Matrix-to-Discord appservice bridge."
+maturity = "Beta"
+language = "Javscript"
+license = "AGPL-3.0-or-later"
+docs = "https://gitdab.com/cadence/out-of-your-element/src/branch/main/docs"
+repo = "https://gitdab.com/cadence/out-of-your-element"
+room = "#out-of-your-element:cadence.moe"
+featured = false
```

Comment:
What does `featured` mean?

Reply:
Featured is a flag we intended to use to promote the projects more heavily, to draw attention to the most sustainable and polished projects. It's only used to add a little "Recommended" chip for now, and is left to the discretion of the m.org crew before we have a more formal process for it :)

You can follow or contribute to https://github.com/matrix-org/matrix.org/issues/1584 if you want!

Reply:
Sweet as. Thank you Thibault!